client/jetbrains/src/main/java/com/sourcegraph/cody/context/CurrentlyOpenedFileListener.java

```diff
@@ -0,0 +1,32 @@
+package com.sourcegraph.cody.context;
+
+import com.intellij.openapi.fileEditor.FileEditorManagerEvent;
+import com.intellij.openapi.fileEditor.FileEditorManagerListener;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.sourcegraph.common.ProjectFileUtils;
+import java.util.Optional;
+import org.jetbrains.annotations.NotNull;
+
+public class CurrentlyOpenedFileListener implements FileEditorManagerListener {
```

Comment:
```suggestion
public class CurrentlyOpenFileListener implements FileEditorManagerListener {
```
https://dictionary.cambridge.org/grammar/british-grammar/open-or-opened

Reply:
thanks, fixed